agent/go.mod

```diff
@@ -95,6 +95,7 @@ require (
 	github.com/json-iterator/go v1.1.12 // indirect
 	github.com/labstack/echo-contrib v0.11.0 // indirect
 	github.com/labstack/gommon v0.4.0 // indirect
+	github.com/lib/pq v1.10.2 // indirect
```

Comment:
I think this change should go away after we change the bun handling

Reply:
Done!